### ISSUE TITLE
Feature request: disable hotkey popup

### ISSUE THREAD

**Issue by jep-dev** [buunguyen/octotree#383]
I'd really like to disable or at least configure the popup. It disappears too fast for me to inspect and restyle, I can't find it in the page source, and I only found ".popup" in the source here (and restyling .popup has no effect.) I tried clearing my hotkeys and clicking 'save', thinking that might disable it, but it refuses to save that.

**Comment by buunguyen** [comment]
Which hotkey? And which disappears too fast?

**Comment by jep-dev** [comment]
"Octotree is enabled on this page. Click this button or press cmd+shift+s (or ctrl+shift+s) to view it." It appears to have a 5-second timer, which should be plenty to find it, but I haven't been able to. The relevant setting (ctrl+shift+s) is what I cleared from the config section. I also did some digging in Keymaster but I couldn't find it there either. Thanks for taking the time to look into this... it only shows up once per session, but if you mess with sessions/profiles/incognito a lot it can get in the way.
